tests/testthat/app.crunch.io/api/account.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Yeah, this is how I was imagining getting around the error in `GET` before the `expect_POST`. It seems like it works for me, do you still get errors? (If so, did you run `make compress-fixtures`?)